packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -159,7 +159,8 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         IERC20 currency,
         uint256 reward
     ) external override nonReentrant() returns (uint256 totalBond) {
-        require(_getState(msg.sender, identifier, timestamp, ancillaryData) == State.Invalid, "requestPrice: Invalid");
```

Comment:
These are minor changes we can make to the `OptimisticOracle` that saves 10,000 gas. I can remove since its probably not worth it.

Reply:
I'd say rather not seeing that would mean re-deploying the OO to get things up to the latest version in production.

Reply:
Reverted